server-plugins/hr-resources/src/index.ts

```diff
@@ -348,17 +356,86 @@ export async function RequestTextPresenter (doc: Doc, control: TriggerControl):
   return `${who} - ${type.toLowerCase()} ${date}`
 }
 
+/**
+ * @public
+ */
+export async function OnPublicHolidayCreate (tx: Tx, control: TriggerControl): Promise<Tx[]> {
+  const actualTx = TxProcessor.extractTx(tx)
+  if (core.class.TxCreateDoc !== actualTx._class) {
+    return []
+  }
+  const ctx = actualTx as TxCreateDoc<PublicHoliday>
+  if (ctx.objectClass !== hr.class.PublicHoliday) {
+    return []
+  }
+
+  const sender = await getEmployeeAccountById(ctx.modifiedBy, control)
+  if (sender === undefined) return []
+  const employee = await getEmployee(sender.employee, control)
+  if (employee === undefined) return []
+
+  const publicHoliday = TxProcessor.createDoc2Doc(ctx)
```

Comment:
We need notification for every subscribers in department, not just one person.

Reply:
This is the user who marked day as public holiday. But also added department into publicHoliday which is sent to email notification where it gets subscribers for department.